.github/workflows/ignore_paths.yml

```diff
@@ -1,6 +1,6 @@
-name: 'ignore_paths'
```

Comment:
Sorry if these yaml formatting changes are undesirable -- I used VSCode for editing and it was pretty opinionated.